### PR TITLE
Adds a file system helper file for determining the file system type.

### PR DESCRIPTION
**Pull request by ehotinger**
@jterry75 as discussed in https://github.com/containerd/containerd/issues/3362

cc @kevpar 

Signed-off-by: Eric Hotinger <ehotinger@gmail.com>

### CODE REVIEW

File: pkg/fs/fs_windows.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	"path/filepath"
+	"syscall"
+	"unsafe"
+
+	"golang.org/x/sys/windows"
+)
+
+// getFileSystemType obtains the type of a file system through GetVolumeInformation.
+// https://msdn.microsoft.com/en-us/library/windows/desktop/aa364993(v=vs.85).aspx
+func getFileSystemType(path string) (fsType string, hr error) {
```
Comment (jterry75) [inline pkg/fs/fs_windows.go]: You need to export this right?

Comment (ehotinger) [reply pkg/fs/fs_windows.go]: Whoops thanks!!

**Comment by jterry75** [comment]
I assume you are going to re-vendor into cd now?

**Comment by ehotinger** [comment]
@jterry75  ya I'll PR it

**Comment by jterry75** [comment]
Thanks!

**Pull request merged**